src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -2246,7 +2246,8 @@ because xbuild doesn't support framework reference assemblies.
     AndroidEmbedProfilers="$(AndroidEmbedProfilers)"
     AndroidGdbDebugServer="$(AndroidGdbDebugServer)"
     HttpClientHandlerType="$(AndroidHttpClientHandlerType)"
-    TlsProvider="$(AndroidTlsProvider)">
+    TlsProvider="$(AndroidTlsProvider)"
+    EnableSGenConc="$(EnableSGenConc)">
```

Comment:
@jonpryor 1. It is not the wisest, but just a way to promote concurrent gc transition, keeping both options available. In very few releases this should go back to the default, which will be concurrent this time.
2. https://github.com/xamarin/md-addins/pull/1407 adds a checkbox in XS
3. This is meant to expose it easily across all platforms mac, ios & android and encourage its use, as opposed to some mysterious environment configurations